GoogleTestAdapter/Core/GoogleTestDiscoverer.cs

```diff
@@ -70,12 +70,12 @@ private static void DiscoverTests(string executable, ITestFrameworkReporter repo
                 Action<TestCase> reportTestCases = tc =>
```

Comment:
Needs to update the legal header at the top of the file to "8/2017".
Apply to all other files with different month there as well.